docs/user/using-the-capture-app.md

```diff
@@ -152,7 +152,8 @@ The second option, is to register a tracked entity instance with program and enr
     the field. This will open a map where you can search for a location and capture
     a polygon (button in the upper right corner of the map).
 
-7. Click **Save new** to register the tracked entity instance.
+7. Click **Save new** to register the tracked entity instance. 
+    The **Cancel** button will be disabled, and a spinner on the **Save** button will indicate that the request has been sent.
```

Comment:
```suggestion
    The **Save** button shows a loading spinner and the **Cancel** button is disabled while the request is processing.
```

Reply:
Thanks for the review @cooper-joe. The docs are now updated with your suggestions.